docs/get-details/dapps/avm/teal/index.md

```diff
@@ -12,8 +12,8 @@ Some of the opcodes in TEAL are only valid for a specific contract type. These a
 !!! warning
     When writing smart contracts, make sure to follow [TEAL guidelines](guidelines). This is very important in order to prevent smart contracts from being compromised.
 
-# TEAL versions
-Currently, Algorand supports versions 1 through 6 of TEAL. When writing contracts with TEAL version 2 or higher, make sure to add `#pragma version #` where # should be replaced by the specific number, as the first line of the program. If this line does not exist, the protocol will treat the contract as a version 1 contract. If upgrading a contract to version 2 or higher, it is important to verify you are checking the `RekeyTo` property of all transactions that are attached to the contract. See the [TEAL guidelines](guidelines) for more suggestions to help prevent a contract from being compromised. 
+# Program versions
+Currently, Algorand supports versions 1 through 6 of TEAL. When writing contracts with Program version 2 or higher, make sure to add `#pragma version #` where # should be replaced by the specific number, as the first line of the program. If this line does not exist, the protocol will treat the contract as a version 1 contract. If upgrading a contract to version 2 or higher, it is important to verify you are checking the `RekeyTo` property of all transactions that are attached to the contract. See the [TEAL guidelines](guidelines) for more suggestions to help prevent a contract from being compromised. 
```

Comment:
@barnjamin Is it possible to link elsewhere to show current AVM versions?

> Algorand supports versions 1 through 6 of TEAL

* I imagine the max version is liable to go stale.  
* I can also imagine you deliberately wish to provide an inline reference.  If that's the case, I leave the following _suggestion_.

```suggestion
Currently, Algorand supports program versions 1 through 6. When writing contracts with Program version 2 or higher, make sure to add `#pragma version #` where # should be replaced by the specific number, as the first line of the program. If this line does not exist, the protocol will treat the contract as a version 1 contract. If upgrading a contract to version 2 or higher, it is important to verify you are checking the `RekeyTo` property of all transactions that are attached to the contract. See the [TEAL guidelines](guidelines) for more suggestions to help prevent a contract from being compromised. 
```

Reply:
I agree we need to find a way to automatically keep this up to date. Either with some script or a link. Will make note and consider it outside this pr